client/genclients.go

```diff
@@ -71,7 +83,10 @@ func (c *Client) WithTimeout(timeout time.Duration) *Client {
 	return c
 }
 
-`, packageName, s.Info.InfoProps.Title)
+`, packageName,
```

Comment:
packageName contains a few extra things as well, but basically yes. I'm hoping to build this into `ark init`.